src/components/OrderItem/OrderItem.jsx

```diff
@@ -11,12 +12,16 @@ export const OrderItem = ({
   bread,
   isHot,
   hasButton,
-  orderOwner
+  orderOwner,
+  ownerId
 }) => {
   const handleClick = () => {
     removeFromOrder(id);
   };
 
+  const isOrderOwner = cookie.load('ownerId') === ownerId || cookie.load('ownerId') === process.env.REACT_APP_SKELETON_KEY;
+  const showButton = isOrderOwner || hasButton;
```

Comment:
Does it need both of these? Its using `isOrderOwner` within `showButton` 🤔 Seems like its repeating some of this logic

Reply:
yeah bit of a weird one this, it's because the modal uses the same `OrderItem` component and I don't want it to show the button in that scenario